doc/scurve-plotting-tools.rst

```diff
@@ -0,0 +1,73 @@
+S-curve plotting tools
+======================
+
+The following tools exist for helping to understand scurve data:
+
+1. :doc:`gemSCurveAnaToolkit.py </man/gemSCurveAnaToolkit>`
+2. :program:`plot_noise_vs_trim.py`
+3. :program:`plot_vfat_and_channel_Scurve.py`
+4. :program:`plot_vfat_summary.py`
+5. :program:`summary_plots.py`
+
+These tools can all by found in the ``macros/`` subdirectory and are designed to
+be run on ``TFile`` objects containing the ``scurveFitTree`` ``TTree`` object
+(e.g. produced by :program:`anaUltraScurve.py`).  The first tool
+:program:`gemSCurveAnaToolkit.py` is for plotting the same (vfat,channel/ROBstr)
+scurve from a list of scandates and it is described in a dedicated subsection
+below. The rest of the tools above are for making plots from a single input
+file; the plots made by tools 2-4 are:
+
+- :program:`plot_noise_vs_trim.py`: Plots a channel/strip's scurve width (e.g.
+  ``noise``) vs. trimDAC as a ``TH2D`` on a ``TCanvas``,
+- :program:`plot_vfat_and_channel_Scurve.py`: Plots a channel/strip's scurve as a
+  ``TH1D`` and its ``TF1`` on a ``TCanvas``, and
+- :program:`plot_vfat_summary.py`: Plots all scurves from a given VFAT as a ``TH2D`` on
+  a ``TCanvas``.
+
+Tool 5 :program:`summary_plots.py` produces the following plots from a single
+input file for a given VFAT depending on the command line argument supplied:
+
+- Plot of channel/strip scurve mean as a ``TH1D``,
+- Plot of channel/strip scurve width as a ``TH1D``,
+- Plot of channel/strip scurve pedestal as a ``TH1D``,
+- Plot of Chi<sup>2</sup> of the channel/strip scurve fits as a ``TH1D``,
+- Plot of channel/strip scurve mean vs. scurve width as a ``TH2D``, and
+- Plot of channel/strip scurve width vs. trimDAC as a ``TH2D``.
+
```

Comment:
should add a description here like:

```
Tool 6 :program:`plotSCurveFitResults.py` ... <summarize your lines 16-22 of macros/plotSCurveFitResults.py, below>
```

Basically how we have for `gemSCurveAnaToolkit.py`

Reply:
still needed

Reply:
Done